Change the required version to suppress the pip check warning. The version of
astroid we have is slightly older but should be fully compatible.

--- pylint-2.15.2/pyproject.toml
+++ pylint-2.15.2/pyproject.toml
@@ -37,7 +37,7 @@ classifiers = [
     # Also upgrade requirements_test_min.txt if you are bumping astroid.
     # Pinned to dev of second minor update to allow editable installs and fix primer issues,
     # see https://github.com/PyCQA/astroid/issues/1341
-    "astroid>=2.12.9,<=2.14.0-dev0",
+    "astroid>=2.12,<=2.14.0-dev0",
     "isort>=4.2.5,<6",
     "mccabe>=0.6,<0.8",
     "tomli>=1.1.0;python_version<'3.11'",
